tests/Microsoft.Identity.Test.Integration.net45/HeadlessTests/OboTests.cs

```diff
@@ -65,6 +70,60 @@ public async Task OBO_WithCache_MultipleUsers_Async()
             await RunOnBehalfOfTestAsync(aadUser2, true).ConfigureAwait(false);
         }
 
+        [TestMethod]
+        [Timeout(2 * 60 * 1000)] // 2 min timeout
+        public async Task OBO_WithCache_MSA_Async()
```

Comment:
>OBO_WithCache_MSA_Async [](start = 26, length = 23)

I was hoping that this would basically be calling an existing test with the only difference being the actual user? Are refactoring's planned?

Reply:
the issue is we can't do ROPC w/msa accounts, so have to do another flow, like device code or something to get an RT. so it's not just a matter of running the same test w/an MSA account.

---
In reply to: [345534322](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/1496#discussion_r345534322) [](ancestors = 345534322)